tests/utils/test_toolset_config.py

```
@@ -117,3 +119,88 @@ def test_new_prometheus_fields_no_warning(self, caplog):
+
+    def test_old_and_new_configs_produce_same_result(self, caplog):
+        """Test that configs created with old and new field names produce identical results."""
+        from holmes.plugins.toolsets.rabbitmq.api import RabbitMQClusterConfig
+
+        with caplog.at_level(logging.WARNING):
+            # Create config using old field names
+            old_config = RabbitMQClusterConfig(
+                id="test-cluster",
+                management_url="http://rabbitmq:15672",
+                username="user",
+                password="pass",
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to argument: "password"

(S106)